slackclient/client.py

```diff
@@ -88,27 +91,29 @@ def api_call(self, method, timeout=None, **kwargs):
         response_body = self.server.api_call(method, timeout=timeout, **kwargs)
         try:
             result = json.loads(response_body)
-        except ValueError as json_decode_error:
-            raise ParseResponseError(response_body, json_decode_error)
-        if self.server:
```

Comment:
just noticed that this check didn't make it over. was it superfluous in the first place? can `self.server` ever not be set?

Reply:
I actually have a stashed patch that removed, I just need to make sure before I commit to it. This is just reordering those operations.

Reply:
@aoberoi I don't think there's a way it _wouldn't_ be set...

It's set when the client is initialized:
https://github.com/slackapi/python-slackclient/blob/master/slackclient/client.py#L30-L33

unless that check is checking this eq 🤔  
https://github.com/slackapi/python-slackclient/blob/master/slackclient/server.py#L35-L39